docs/map-making/upgrading-1-8-maps.md

```diff
@@ -1,88 +1,147 @@
 A list of (case sensitive) changes, to upgrade a map from a TripleA version to another one.
 
 ## Upgrade from 1.8.0.9 to 1.9.0.0.3266
-(old:new)
+
 Note: The following list covers all basic upgrade cases for maps, but it may not upgrade "Property" options (playerProperty, unitProperty...) of already deprecated items.
+
 Note: Before applying the changes below (assuming you do it with a simple replace), you should make sure to not have unnecessary spaces inside angle brackets (chevrons).
-Note: Some items are not validated; thus, if your game plays on 1.9.0.0.3266, this doesn't necessarily mean that you correctly upgraded it.
 
-### Set:
+Note: Some items are not validated; thus, if your game plays on 1.9.0.0.3266, this doesn't necessarily mean that you correctly upgraded it (for more information, see the "warning" section).
+
+### New XML settings:
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
+
+`<triplea minimumVersion="*"/>`:`<triplea minimumVersion="1.9.0.0"/>`
+
+(The "*" means whatever)
+
+### Not already deprecated XML changes:
 
-`<triplea minimumVersion="1.9.0.0"/>`
+_(these are the main changes on game file items that were not deprecated, so you normally need to change these only, within the game file)_
 
-### Not already deprecated xml changes:
-_(you normally need to change these only)_
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
 
 `attatchment`:`attachment`
+
 `Attatchment`:`Attachment`
+
 `attatchTo=`:`attachTo=`
+
 `="isImpassible"`:`="isImpassable"`
+
 `="conditionType" value="XOR"`:`="conditionType" value="1"`
-`="turns":="rounds"`
+
+`="turns"`:`="rounds"`
+
 `="Battleships repair at end of round"`:`="Units Repair Hits End Turn"`
+
 `="Battleships repair at beginning of round"`:`="Units Repair Hits Start Turn"`
+
 `="Naval Bombard Casualties Return Fire Restricted"`:`="Naval Bombard Casualties Return Fire"`
 
-### Special code xml changes for the above changes:
-_(you probably don't have these)_
+### Special code XML changes for the above changes:
+
+_(these are cases of advanced coding, so you probably don't have these)_
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
```

Comment:
I wonder if we can state this just once somewhere at the beginning of the document. Keeping things short and concise is valuable and makes it take less time to use a document (which makes it more useful).

Reply:
Not really. There are some items that are not game file (XML) codes and that are not changes (you could present them with an empty "new" version, to mean that they need to be removed (substituted with nothing), but that would be unclear).

For the format of "old" and "new" set apart by a colon ("change all occurrences of the code before the colon with the code after the colon"), that could be specified once (and that is what I was doing before this set of changes), but then I feel like the phrase where you tell the specific stuff is hardly meaningful.